src/api/checkout/checkout_payment_activations_auth/v1/_swagger.json.tpl

```diff
@@ -39,15 +39,33 @@
             }
           },
           "400": {
-            "description": "Invalid input or PagoPA response",
+            "description": "Invalid input",
```

Comment:
v1 auth APIs do not need to be updated, because used by IO. The goal of this PR is to update only the auth v2 specs and ensure compatibility with auth v1 and checkout api v1 (with apim policy)

Reply:
cc @giovanniberti